server/config.js

```diff
@@ -10,6 +10,7 @@ var conf = module.exports = {
     port: process.env.PORT || 3001
   },
   pg: {
+    url: process.env.DATABASE_URL || null,
```

Comment:
No reason to add ` || null` here. I guess you copied it from the `redis` property, but it shouldn't have been there either